packages/office-ui-fabric-react/src/components/Persona/Persona.tsx

```diff
@@ -66,7 +66,9 @@ export class Persona extends React.Component<IPersonaProps, any> {
 
     let divProps = getNativeProps(this.props, divProperties);
     let personaDetails = <div className='ms-Persona-details'>
-      <div className='ms-Persona-primaryText'>{ primaryText }</div>
+      <div className='ms-Persona-primaryText'>
+        <div className='ms-Persona-primaryTextContent'>{ primaryText }</div>
```

Comment:
Why do we need an extra div here? We should be able to accomplish the same thing with a single div. Rarely should elements need to have a single child, we can just roll the child's styles into its parent's styles.